velero-plugin-for-microsoft-azure/object_store.go

```diff
@@ -136,47 +136,51 @@ func newObjectStore(logger logrus.FieldLogger) *ObjectStore {
 }
 
 func getStorageAccountKey(config map[string]string) (string, *azure.Environment, error) {
-	// load environment vars from $AZURE_CREDENTIALS_FILE, if it exists
+	// // load environment vars from $AZURE_CREDENTIALS_FILE, if it exists
```

Comment:
looks like we've got a bunch of double-comments here (`// //`) - can you clear those?